__docs__/wonder-blocks-dropdown/single-select.stories.tsx

```diff
@@ -125,6 +126,7 @@ const styles = StyleSheet.create({
         padding: Spacing.medium_16,
     },
     focused: {
+        backgroundColor: mix(Color.lightBlue, Color.offWhite),
```

Comment:
**Question**: Is this focused style being applied? I do see the outline, but I'm not sure I see a different background color when I focus on the custom opener in the [story](https://5e1bf4b385e3fb0020b7073c-yxpybycqjl.chromatic.com/?path=/story/dropdown-singleselect--custom-opener).

Reply:
hmm yeah, I was experimenting with it but forgot to get rid of this. I'll do it before landing.